src/map/status.c

```diff
@@ -2259,6 +2264,8 @@ int status_calc_pc_(struct map_session_data* sd, enum e_status_calc_opt opt) {
 	//Give them all modes except these (useful for clones)
 	bstatus->mode = MD_MASK&~(MD_BOSS|MD_PLANT|MD_DETECTOR|MD_ANGRY|MD_TARGETWEAK);
 
+	status->calc_pc_additional(sd,opt);
```

Comment:
It is declared in the .h file, so it will work. And even if it is never initialized, the C specs guarantee that it is initialized to 0 / NULL. (although, it might be a good idea to explicitly initialize it to NULL in `status_defaults`, for documentation purposes, rather than just removing the line)